lib/Controller/User.php

```diff
@@ -1524,6 +1524,116 @@ public function permissionsGrid(Request $request, Response $response,$entity, $i
         return $this->render($request,  $response);
     }
 
+
+    /**
+     * @SWG\Get(
+     *  path="/user/permissions/{entity}",
+     *  operationId="userPermissionsMultiSearch",
+     *  tags={"user"},
+     *  summary="Permission Data",
+     *  description="Permission data for the multiple Entities and Objects Provided.",
+     *  @SWG\Parameter(
+     *      name="entity",
+     *      in="path",
+     *      description="The Entity",
+     *      type="string",
+     *      required=true
+     *   ),
+     *  @SWG\Parameter(
+     *      name="ids",
+     *      in="query",
+     *      description="The IDs of the Objects to return permissions for",
+     *      type="string",
+     *      required=true
+     *   ),
+     *  @SWG\Response(
+     *      response=200,
+     *      description="successful operation",
+     *      @SWG\Schema(
+     *          type="array",
+     *          @SWG\Items(ref="#/definitions/Permission")
+     *      )
+     *  )
+     * )
+     *
+     * @param Request $request
+     * @param Response $response
+     * @param string $entities
+     * @return \Psr\Http\Message\ResponseInterface|Response
+     * @throws AccessDeniedException
+     * @throws GeneralException
+     * @throws \Xibo\Support\Exception\ControllerNotImplemented
+     * @throws \Xibo\Support\Exception\NotFoundException
+     */
+    public function permissionsMultiGrid(Request $request, Response $response, $entity)
+    {
+        $sanitizedParams = $this->getSanitizer($request->getParams());
+
+        // Check if the array of ids is passed
+        if($sanitizedParams->getString('ids') == "") {
+            throw new InvalidArgumentException(__('The array of ids is empty!'));
+        }
+
+        // Get array of ids
+        $ids = explode(',', $sanitizedParams->getString('ids'));
+
+        // Array of all the permissions
+        $permissions = array();
```

Comment:
We should try to use the newer array syntax for new code - so `[]` instead of `array()`